crates/node/Cargo.toml

```diff
@@ -19,6 +19,7 @@ serde_json.workspace = true
 tokio = { workspace = true, features = ["io-std", "macros"] }
 tracing.workspace = true
 url.workspace = true
+webbrowser = "0.8"
```

Comment:
1. please move this version association to the root `Cargo.toml`, and only keep `webbrowser.workspace = true` here, like everything else
2. `webbrowser v0.8` is over 2 years old, did you have any issues compiling v1.0.4?